ci-operator/config/openshift/release/openshift-release-master__okd-4.13.yaml

```diff
@@ -43,14 +50,32 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-ovn
```

Comment:
Lets leave `ovn` prefix - this is a requirement from Sippy folks to distinguish SDN/OVN jobs